qa/suites/rados/cephadm/dashboard/0-distro/.qa

```diff
@@ -0,0 +1 @@
+../.qa
```

Comment:
this change actually **adds** a new test suite to `qa/suites/rados/cephadm/dashboard`. is this intended?

Reply:
@tchaikov I addressed most of the issues with PR and also removed some of the additional suites that were introduced with cherrypicks, added them in trivial fix comment in commits. Can you help review, thanks!